### PR TITLE
Fetch requirements directly from openvino tools

### PR DESCRIPTION
**Pull request by slyubimt**
### Details:
 - *Enable extraction list of requirements for Accuracy Checker, POT, Benchmark*

### CODE REVIEW

File: tools/openvino_dev/requirements.txt [hunk 1]
```diff
@@ -1,28 +1,7 @@
...
-pydicom>=2.1.2
-sentencepiece>=0.1.95
-tokenizers>=0.10.1
-editdistance>=0.5.3
-parasail>=1.2.4
-fast-ctc-decode>=0.2.5
-rawpy>=0.16.0
-nltk>=3.5
 opencv-python==4.5.*
 progress>=1.5
```
Comment (ilya-lavrenov) [inline tools/openvino_dev/requirements.txt:28]: What is the origin of other requirements?

Comment (eaidova) [reply tools/openvino_dev/requirements.txt:28]: benchnmark_app and pot

File: tools/openvino_dev/requirements.txt [hunk 2]
```diff
@@ -1,28 +1,7 @@
...
 hyperopt~=0.1.2
-networkx~=2.5
-tqdm>=4.54.1
 texttable~=1.6.3
-py-cpuinfo>=7.0.0
-PyYAML>=5.4.1
-pillow>=8.1.2
-scikit-image>=0.17.2
-scikit-learn>=0.24.1
 yamlloader>=0.5
```
Comment (eaidova) [inline tools/openvino_dev/requirements.txt:5]: please consider opportunity to remove yamlloader (it is not listed in other components requirements)

**Review by eaidova — APPROVED** [review]

File: tools/openvino_dev/requirements.txt (hunk 1 above)
Comment (slyubimt) [reply tools/openvino_dev/requirements.txt:28]: cleaned up this list to get all requirements from tools directly

File: tools/openvino_dev/requirements.txt (hunk 2 above)
Comment (slyubimt) [reply tools/openvino_dev/requirements.txt]: done

**Comment by slyubimt** [comment]
Depend on https://github.com/openvinotoolkit/openvino/pull/8740

File: tools/openvino_dev/requirements.txt (hunk 1 above)
Comment (ilya-lavrenov) [reply tools/openvino_dev/requirements.txt:28]: this file is empty now, can we drop it?

Comment (slyubimt) [reply tools/openvino_dev/requirements.txt:28]: we have plans to add other dependencies, like telemetry

Comment (ilya-lavrenov) [reply tools/openvino_dev/requirements.txt:28]: why is it a common dependency? It should be dependency of particular tool anyway

**Pull request merged**